src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -199,8 +199,11 @@ static TexlLexer()
         public static TexlLexer GetLocalizedInstance(CultureInfo culture)
         {
             culture ??= CultureInfo.InvariantCulture;
+
+            // Number decimal separator can be a dot (.), comma (,), arabic comma (Unicode 0x66B) 
+            // .Net core 3.1 doesn't support arabic comma and defines '/' as decimal separator for Persian/Farsi (fa, fa-IR) which is probably a bug
```

Comment:
>// [](http://example.com/codeflow?start=12&length=2)

Interesting ... Is there something to followup here?
Can we craft a bug

Reply:
Waiting on Carlos's thread he initiated with another team to know if we will soon support RTL languages like Arabic and Hebrew